app-office/gnucash/gnucash-4.12.ebuild

```diff
@@ -74,40 +76,56 @@ RDEPEND="
 	)
 "
 
-DEPEND="${RDEPEND}
-	>=dev-cpp/gtest-1.8.0
+DEPEND="
+	${RDEPEND}
 	>=sys-devel/gettext-0.20
 	dev-lang/perl
 	dev-perl/XML-Parser
 	sys-devel/libtool
+	test? ( >=dev-cpp/gtest-1.8.0 )
 "
 
 BDEPEND="
 	dev-lang/swig
 	dev-util/cmake
 	virtual/pkgconfig
+	|| (
```

Comment:
Why?

Reply:
That's not made by me. It was already like that